src/com/owncloud/android/ui/activity/ImageSimpleAdapter.java

```diff
@@ -0,0 +1,123 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   @author Tobias Kaminsky
+ *   Copyright (C) 2016  Tobias Kaminsky
```

Comment:
The Copyright of the project belongs to OwnCloud (I suppose that you know because in the second point of the Contributor Agreement
https://owncloud.org/wp-content/themes/owncloudorgnew/assets/files/owncloud-copyright-assignment-common.pdf). 
So, could you please remo remove this Copyright line?
The @author line is ok.